### ISSUE TITLE
Large number of `looking at node` entries in the tsserver log

### ISSUE THREAD

**Issue by mjbvz** [MicrosoftDocs/intellicode#103]
**Product and Version** VS COde
**OS Version** [macOS/Windows]: NA
**IntelliCode Extension Version**: 1.1.6 
**Target Platform or Language**  TS

**Steps to Reproduce / Scenario:**

1. In VS Code, set `"typescript.tsserver.log": "normal"`
1. With intellicode enabled, open a ts file
1. Then run the `Open TS Server log` command in VS Code. Look at the opened tsserver log file

**Bug**
Notice a huge number of ` looking at node` log entries in the log.

```
Info 121  [16:2:14.463] program found
Info 122  [16:2:14.463] source file found
Info 123  [16:2:14.463] looking at node 250 353 439 11580: import { OnBeforeRequestDetails, OnHeadersReceivedDetails, Response } from 'electron';
Info 124  [16:2:14.463] looking at node 251 360 422 11580: { OnBeforeRequestDetails, OnHeadersReceivedDetails, Response }
Info 125  [16:2:14.463] looking at node 253 360 422 11580: { OnBeforeRequestDetails, OnHeadersReceivedDetails, Response }
Info 126  [16:2:14.463] looking at node 254 362 384 11580: OnBeforeRequestDetails
Info 127  [16:2:14.463] looking at node 73 362 384 11580: OnBeforeRequestDetails
Info 128  [16:2:14.463] looking at node 254 386 410 11580: OnHeadersReceivedDetails
Info 129  [16:2:14.463] looking at node 73 386 410 11580: OnHeadersReceivedDetails
Info 130  [16:2:14.463] looking at node 254 412 420 11580: Response
Info 131  [16:2:14.463] looking at node 73 412 420 11580: Response
...
```

After checking with the TS team, we believe these logged lines are coming from IntelliCode. The volume of data being logged makes the log file difficult for us to analyze and may be impacting performance.

Please:

* Disable this logging (perhaps except in verbose logging mode, but even then it's probably too verbose)
* Make sure any log messages are prefixed with `intellicode` or some other identifiable prefix in the future

**Comment by jkeech** [comment]
Thanks for reporting. Yes, this is coming from the IntelliCode plugin. I agree with the proposed solution. @uniqueiniquity, can you fix this?

**Comment by jkeech** [comment]
This has been fixed and will ship in 1.1.7.
